highfidelity/hifi
# Tablet extended sounds

https://highfidelity.fogbugz.com/f/cases/6613/Add-Sound-Cues-to-OnHover-and-Click-Behavior

Test plan:
- open the tablet. Should have a sound emitted.
- Highlight over buttons, checkboxes, menu items, etc. Notice hover sounds
- Click on buttons, checkboxes, menu items, etc. Notice click sounds.
